File: basecommon/src/main/java/com/common/http/bean/ExampleListBean.java
package com.common.http.bean;

/**
 * @Author: 张鹏飞
 * @Email: 1271396448@qq.com
 * @Date: 2020/11/18
 * <p>
 * @Desc: 集合例子Bean
 */
public class ExampleListBean {

    private int id;
    private String title;
    private String content;
    private String time;

    /**
     * 多选用到,如果选中就设为true,未选中就设置为false
     */
    private boolean checked;

    public boolean isChecked() {
        return checked;
    }

    public void setChecked(boolean checked) {
        this.checked = checked;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }
}

File: basecommon/src/main/java/com/common/http/bean/ExampleListBean.kt
package com.common.http.bean

/**
 * @Author: 张鹏飞
 * @Email: 1271396448@qq.com
 * @Date: 2020/11/18
 *
 *
 * @Desc: 集合例子Bean
 */
class ExampleListBean {
    var id = 0
    var title: String? = null
    var content: String? = null
    var time: String? = null
    /**
     * 多选用到,如果选中就设为true,未选中就设置为false
     */
    var isChecked = false

}